wordpress-mobile/WordPress-Android
# Minor fixes in RtL mode

Partially fixes #2278
Reverse order of bars in the graph on the stats screen (RtL mode) - time goes from right to left in countries with RtL locale.

Fix homeAsUp arrow direction in RtL mode (login flow SDK < 23)

To test:
SDK <= 22
1. Change locale to Hebrew.
2. Go to the email/site login screen -> the up arrow should be pointing to the right.
3. Go to Statistics - the most recent day (on the X axis) should be display on the left side of the screen (and should be selected).
